framework/db/schema/pgsql/CPgsqlSchema.php

```diff
@@ -166,7 +166,12 @@ protected function resolveTableNames($table,$name)
 	protected function findColumns($table)
 	{
 		$sql=<<<EOD
-SELECT a.attname, LOWER(format_type(a.atttypid, a.atttypmod)) AS type, d.adsrc, a.attnotnull, a.atthasdef,
+SELECT
+	a.attname,
+	LOWER(format_type(a.atttypid, a.atttypmod)) AS type,
+	pg_get_expr(adbin, adrelid) AS default_value_text,
```

Comment:
keeping the name `adsrc` here would make this fully BC save, in case someone extended that class.

```suggestion
	pg_get_expr(adbin, adrelid) AS adsrc,
```

Reply:
Added to this PR